### DISCUSSION TITLE
Transcribing translates audio into English.

### DISCUSSION THREAD

**Discussion by evylegzhanin** [openai/whisper#186] [Q&A] [answered]
I have an audio file with a speech in Russian. Somehow the whisper (large model) detects the English language and using the basic parameters the model automatically translates all speech from Russian into English when transcribing. If I specify the language as Russian than the model skips the first 15-25 seconds of the audio. The attempts of manually specifying the task as 'transcribe' didn't help either. Any ideas how to fix it would be appreciated. Thanks!

**Comment by jongwook** [comment, marked as answer]
Looks like our language detector was not accurate enough, and specifying Russian is the intended approach in this case. Tweaking the `max_initial_timestamp` option may help in this case, but otherwise I'm not sure how exactly we can fix it. If you can upload/email the audio file, I can try looking into it.

**Reply by evylegzhanin** [reply]
Thank you a lot for your response! I've sent you an e-mail.
